apps/web/src/views/TradingReward/components/TopTraders/CurrentRewardPool.tsx

```diff
@@ -0,0 +1,189 @@
+          </Text>
+        </Flex>
+        <Flex justifyContent="space-between" mb="10px">
+          <Text color="white" fontWeight={['400', '400', '400', '600']} fontSize={['14px', '14px', '14px', '20px']}>
+            {t('Ends')}
+          </Text>
+          {timeRemaining > 0 ? (
+            <Text bold color="white" fontSize={['14px', '14px', '14px', '20px']}>
+              {t('in')}
+              {timeUntil.months ? (
+                <Text bold color="white" fontSize={['14px', '14px', '14px', '20px']} as="span" ml="4px">
+                  {`${timeUntil.months}${t('m')}`}
+                </Text>
+              ) : null}
+              {timeUntil.days ? (
+                <Text bold color="white" fontSize={['14px', '14px', '14px', '20px']} as="span" ml="4px">
+                  {`${timeUntil.days}${t('d')}`}
+                </Text>
+              ) : null}
+              {timeUntil.days || timeUntil.hours ? (
+                <Text bold color="white" fontSize={['14px', '14px', '14px', '20px']} as="span" ml="4px">
+                  {`${timeUntil.hours}${t('h')}`}
+                </Text>
+              ) : null}
+              <Text bold color="white" fontSize={['14px', '14px', '14px', '20px']} as="span" ml="4px">
+                {`${timeUntil.minutes}${t('m')}`}
+              </Text>
+            </Text>
+          ) : (
+            <Text bold color="white" fontSize={['14px', '14px', '14px', '20px']}>
+              {timeFormat(locale, incentives?.campaignClaimTime)}
+            </Text>
+          )}
+        </Flex>
+        <Flex justifyContent="space-between" mb="10px">
+          <Text color="white" fontWeight={['400', '400', '400', '600']} fontSize={['14px', '14px', '14px', '20px']}>
+            {t('To win')}
+          </Text>
+          <Text bold color="white" fontSize={['14px', '14px', '14px', '20px']}>
+            {t('Rank #500 or higher')}
+          </Text>
+        </Flex>
+        <Flex justifyContent="space-between" mb="10px">
+          <Text color="white" fontWeight={['400', '400', '400', '600']} fontSize={['14px', '14px', '14px', '20px']}>
+            {t('Rewarding')}
+          </Text>
+          <Text bold color="white" fontSize={['14px', '14px', '14px', '20px']}>
+            {t('3% of the trading fee')}
+          </Text>
+        </Flex>
+        <Flex justifyContent="space-between" mb="10px">
+          <Text color="white" fontWeight={['400', '400', '400', '600']} fontSize={['14px', '14px', '14px', '20px']}>
+            {t('Total volume generated')}
+          </Text>
+          <Text bold color="white" fontSize={['14px', '14px', '14px', '20px']}>
+            {`$${formatNumber(campaignInfoData?.totalVolume, 3, 3)}`}
+          </Text>
+        </Flex>
+        <Flex justifyContent="space-between" mb="10px">
+          <Text color="white" fontWeight={['400', '400', '400', '600']} fontSize={['14px', '14px', '14px', '20px']}>
```

Comment:
The `Text` component looks repetitive. Can we refactor into using one component so we don't need to define the same props all over the place?